code/languages/c/static_memory_allocation/example.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+void func(void);
+ 
+static int count = 5; /* global variable */
+
+int  
+main()
```

Comment:
Change to `int main(void)` - This makes it more clear. In C (unlike cpp), if a function signature doesn’t specify any argument, it means that the function can be called with any number of parameters or without any parameters. So this means,  if you define it as `int main()` you can call main with no parameters or any number of parameters.

Reply:
done!